src/exposition/http.rs

```diff
@@ -281,3 +259,55 @@ mod handlers {
         }
     }
 }
+
+// gathers up the metrics into a simple format that can be presented as human
+// readable metrics or transformed into JSON
+async fn simple_stats(quoted: bool) -> Vec<String> {
+    let mut data = Vec::new();
+
+    let snapshots = crate::SNAPSHOTS.read().await;
+
+    let q = if quoted { "\"" } else { "" };
+
+    for metric in &metriken::metrics() {
```

Comment:
Is there any reason not to use the `match metric.value() { Some(Value::Counter(value)) .. ` pattern here?

I know it probably doesn't make a huge difference since we'd chatted about how the metrics are in-crate, but seems like we may as well make over the switch-over if we are making the change.

Reply:
Leaving this open for a follow-up PR.